proto/settlus/oracle/v1alpha1/tx.proto

```diff
@@ -25,6 +25,7 @@ message MsgPrevote {
   string feeder = 1;
   string validator = 2;
   string hash = 3;
+  uint64 round_id = 4;
```

Comment:
what benefit do we get from including `round_id` in the `Prevote` and `Vote` msgs?

Reply:
If a voter votes late, the vote for the last round can be considered as a vote for the current round.

Reply:
I can't think of a scenario where a late vote would get accepted in the message server. 

Let's say a validator submitted `MsgVote` after the voting period has just ended. If a voting period is over, `x/oracle` immediately starts a prevote period. The late vote would still get denied because the `x/oracle` module is now only accepting `MsgPrevote`. 

I think it would be easier if we just deny late votes. If we start considering for late votes, the voting process might get too complicated or have unwanted side-effects.